src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/debugImage/address.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import {DebugImage} from 'app/components/events/interfaces/debugMeta/types';
+import {formatAddress, getImageRange} from 'app/components/events/interfaces/utils';
+import {Image} from 'app/types/debugImage';
+
+const IMAGE_ADDR_LEN = 12;
+
+type Props = {
+  image: Image;
+};
+
+function Address({image}: Props) {
+  const [startAddress, endAddress] = getImageRange(image as DebugImage);
+
+  if (startAddress && endAddress) {
+    return (
+      <React.Fragment>
+        <span>{formatAddress(startAddress, IMAGE_ADDR_LEN)}</span>
+        {' \u2013 '}
+        <span>{formatAddress(endAddress, IMAGE_ADDR_LEN)}</span>
+      </React.Fragment>
+    );
+  }
+
+  return null;
```

Comment:
Can you re-order this logic so that we return the components as the "default" case? It would remove a level of indentation.